### DISCUSSION TITLE
Upload your own custom icon for an entry instead of using favicon

### DISCUSSION THREAD

**Discussion by Tesla-Tank** [keepassxreboot/keepassxc#11604] [Q&A] [answered]
I wanted to know if there was a way to upload a custom icon into the data for entries that can't find one or don't have one to download at all.
 
If it's not possible to do, would it be a good idea or would there be any security risks to uploading a custom icon?

**Comment by droidmonkey** [comment, marked as answer]
Go to the icon tab when editing an entry and choose "Choose icon"

https://keepassxc.org/docs/KeePassXC_UserGuide#_icons

**Reply by Tesla-Tank** [reply]
Thank you very much, I didnt notice it the first time.
